cdap-ui/templates/common/Script.json

```diff
@@ -7,7 +7,6 @@
        "fields" : {
           "script" : {
              "widget": "javascript-editor",
-             "label" : "Script",
```

Comment:
I recall @ajainarayanan mentioning that its not necessary.
Can you clarify, @ajainarayanan ?